docs/reference/components/sources/kafka.cue

```diff
@@ -83,6 +83,36 @@ components: sources: kafka: {
 				examples: ["message_key"]
 			}
 		}
+		topic_key: {
+			common:      true
+			description: "The log field name to use for the Kafka topic key. If unspecified, the key would not be added to the log event."
+			required:    false
+			warnings: []
+			type: string: {
+				default: null
+				examples: ["topic_key"]
```

Comment:
I might just use `topic`, `partition`, etc. as the examples. I think `key_field`'s example was `message_key` because `key` could be ambiguous (I guess?).

Reply:
Sounds good